Framework/HistogramData/inc/MantidHistogramData/Exception.h

```diff
@@ -0,0 +1,16 @@
+#include "MantidHistogramData/DllConfig.h"
+#include <stdexcept>
+
+namespace Mantid {
+namespace HistogramData {
+namespace Exception {
+
+/// Thrown whenever invalid bin edges are encountered by HistogramData Rebin
+class MANTID_HISTOGRAMDATA_DLL InvalidBinEdgesError
+    : public std::runtime_error {
+public:
+  InvalidBinEdgesError(const char *what);
```

Comment:
In C++11 there is a nice way to avoid this trivial implementation:

```cpp
  using std::runtime_error::runtime_error;
```
i.e., you can just bring the base class constructors into scope.